### DISCUSSION TITLE
Entities Replaced with New Entities after Firmware Update via HASS

### DISCUSSION THREAD

**Discussion by barndawgie** [zwave-js/zwave-js#7006] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I updated my Aeotec ZWave Plug via Home Assistant Update entity. When it came back up after successful upgrade, all existing entities were unavailable and new entities suffixed with “_2” had been created.

What did you expect to happen instead?
Existing entities should still have been active

  1. Click on update notification in Home Assistant and choose “Install”

Anything else we should know?

### Software versions

Driver (node-zwave-js): 12.12.4

Z-Wave JS UI: 9.14.6

Home Assistant Z-Wave Integration: 2024.7.2

If you are using something non-standard, tell us here: ZWave-js-ui running in docker

### Device information

Manufacturer: Aeotec
Model name: Smart Switch 7
Node ID: 009

### Checklist

- [ ] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Reply by barndawgie** [reply]
> 👋 Hey @barndawgie!
> 
> Thanks for opening an issue! It doesn't look like you provided a logfile though. While not strictly necessary for every issue, having a **driver log** on loglevel **debug** is required to diagnose most issues.
> 
> Please consider uploading a logfile that captures your problem. As a reminder, here's how to create one:
> 
> - [Z-Wave JS  UI](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs)
> - [Home Assistant Z-Wave Integration](https://www.home-assistant.io/integrations/zwave_js#how-do-i-access-the-z-wave-logs)
> - [ioBroker.zwave2 Adapter](https://github.com/AlCalzone/ioBroker.zwave2/blob/master/docs/en/troubleshooting.md#providing-the-necessary-information-for-an-issue)
> 

I can't reproduce the issue (since the device is already updated) so not sure a log file would be helpful.

**Comment by barndawgie** [comment]
I couldn't get a log of this happening, but Device Diagnostics are attached and you can see the new, "_2" entities that were created.
[zwave_js-d59fb5459a3263b716c27bc840582581-Hot Water Circulator-dc7e1a6217caf91ba610d5252ab8e911.json](https://github.com/user-attachments/files/16228054/zwave_js-d59fb5459a3263b716c27bc840582581-Hot.Water.Circulator-dc7e1a6217caf91ba610d5252ab8e911.json)

**Comment by barndawgie** [comment]
Is there a way to delete the device in Home Assistant an force it to be recreated?

**Comment by AlCalzone** [comment]
It looks like the old values reference endpoint 0 while the new ones reference endpoint 1. To know why, please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs) of a re-interview, loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Comment by barndawgie** [comment]
@AlCalzone - Thanks for getting back to me. Driver Log is attached.
[z-ui_2024-07-15.log](https://github.com/user-attachments/files/16238082/z-ui_2024-07-15.log)
